README.md

```diff
@@ -31,10 +31,12 @@ Once the dependencies are installed, you can build AIKIDO using [CMake]:
 ```shell
 $ mkdir build
 $ cd build
+$ source /opt/ros/indigo/setup.bash
```

Comment:
> ...devel is added), right?

No. `/opt/ros/indigo` is an install space by itself. There is no requirement that devel-spaces live in a directory called "devel" or install spaces live in a directory called "install" - those are just the default name of the directories used by `catkin_make` and `catkin build`.